### DISCUSSION TITLE
新建知识库连接embeds模型，使用的是千问的text-embedding-v2，为什么会是404，是不是千问的路径应该是https://dashscope.aliyuncs.com/api/v1/services/embeddings/text-embedding/text-embedding？

### DISCUSSION THREAD

**Discussion by shuijiaole** [chatchat-space/Langchain-Chatchat#4378] [Q&A]
2024-07-02 16:07:53,025 httpx        10700 INFO     HTTP Request: POST https://dashscope.aliyuncs.com/compatible-mode/v1/embeddings "HTTP/1.1 404 Not Found"

**Comment by cy1019xx** [comment]
对，应该是：https://dashscope.aliyuncs.com/api/v1/services/embeddings/text-embedding/text-embedding
详见：https://help.aliyun.com/zh/dashscope/developer-reference/text-embedding-api-details?spm=a2c4g.11186623.0.0.66223bc98NdlgU
